astropy/modeling/core.py

```diff
@@ -1119,6 +1122,15 @@ def has_user_bounding_box(self):
 
         return self._user_bounding_box is not None
 
+    @property
+    def separable(self):
+        if self._separable is not None:
+            return self._separable
+        else:
+            raise NotImplementedError(
+            'The "separable" property is not defined for '
```

Comment:
Text needs to be indented more. Not sure why PEP8 check didn't complain about this one. Is there a tab in there that I can't see here?